package-lock.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I strongly suspect that checking `package-lock.json` and `package.json` in here were unintentional (probably the result of running `npm install prettier` or similar). If I'm right about that, could you please remove them and add new rules to `.gitignore` to prevent that happening again?